offchain-modules/src/packages/db/entity/EosLock.ts

```diff
@@ -0,0 +1,38 @@
+import { Entity, Column, PrimaryColumn, CreateDateColumn, UpdateDateColumn, Index } from 'typeorm';
+
+@Entity()
+export class EosLock {
+  @PrimaryColumn()
+  @Index()
```

Comment:
Is this redundant since primary key is already an index?

Reply:
fixed